dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard_progress_v2.feature

```diff
@@ -33,6 +33,80 @@ Scenario: Teacher can open and close Icon Key and details
   And I click selector "#ui-close-dialog"
   And element "h3:contains(Progress Tracking Icon Key)" is hidden
 
+Scenario: Teacher can open and close lessons and see level data cells
+  Given I create an authorized teacher-associated student named "Sally"
+  Given I am assigned to unit "allthethings"
+
+  When I sign in as "Teacher_Sally" and go home
+  And I get levelbuilder access
+  When I click selector "a:contains(Untitled Section)" once I see it to load a new page
+  And I wait until element "#uitest-teacher-dashboard-nav" is visible
+  And check that the URL contains "/teacher_dashboard/sections/"
+  And I wait until element "#uitest-course-dropdown" is visible
+  Then I append "/?enableExperiments=section_progress_v2" to the URL
+
+  # toggle to V2 progress view
+  Then I click selector "#ui-test-toggle-progress-view"
+  And element "#ui-test-progress-table-v2" is visible 
+
+  # Teacher can open lesson to view level data
+  And I wait until element "#ui-test-lesson-header-2" is visible
+  And I click selector "#ui-test-lesson-header-2"
+  And I wait until element ".ui-test-1" is visible
+
+  # Teacher can close lesson so level data is no longer visible
+  And I click selector "#ui-test-expanded-progress-column-header-2"
+  And element ".ui-test-1" is not visible
+
+Scenario: Teacher can navigate to student work by clicking level cell.
+  Given I create an authorized teacher-associated student named "Sally"
+  Given I am assigned to unit "allthethings"
+
+  When I sign in as "Teacher_Sally" and go home
+  And I get levelbuilder access
+  When I click selector "a:contains(Untitled Section)" once I see it to load a new page
+  And I wait until element "#uitest-teacher-dashboard-nav" is visible
+  And check that the URL contains "/teacher_dashboard/sections/"
+  And I wait until element "#uitest-course-dropdown" is visible
+  Then I append "/?enableExperiments=section_progress_v2" to the URL
+
+  # toggle to V2 progress view
+  Then I click selector "#ui-test-toggle-progress-view"
+  And element "#ui-test-progress-table-v2" is visible 
```

Comment:
Because lines 62-75 (or some subset) are used a bunch, could we make a new step for them? Maybe in a new file in `dashboard/test/ui/features/step_definitions/` or in `section_management_steps.rb`.

Reply:
Added!